docker/env.example

```diff
@@ -0,0 +1,43 @@
+
+### LLM ###
+
+OPENAI_API_KEY=
+ANTHROPIC_API_KEY=
+
+###  Auth providers ###
+
+GITHUB_CLIENT_ID=
```

Comment:
is it a problem that these will be defined but empty? Should we comment them all out in the example?